app/components/cases/edit-case-with-confidential.hbs

```diff
@@ -0,0 +1,59 @@
+<Auk::Modal>
```

Comment:
Ik zie nu pas dat er al een `cases/edit-case` component was. Dan kan beter deze gebruikt en gerefactored worden naar een DDAU-component. Die component heeft geen toggle voor de vertrouwelijkheid van het dossier, maar dat lijkt dan een dataveld dat de gebruikers niet moeten kunnen wijzigen. 

Ik zou de edit-case-with-confidential component dus terug verwijderen uit deze PR en de edit-case component refactoren. Voor de refactor zal je waarschijnlijk wel grote delen van deze edit-case-with-confidential component  kunnen gebruiken.

De edit-case component wordt ook gebruikt in de `cases/overview` route, dus daar moeten mogelijk ook enkele wijzigingen gebeuren als de component gerefactored wordt naar DDAU.